packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -299,10 +297,13 @@ async function postToSidechainConverter(payload: string, patientId: string, log:
           "x-api-key": apiKey,
         },
       });
+      if (!res.data || !res.data.resourceType || !(res.data.resourceType === "Bundle")) {
+        throw new Error(notFHIRBundleError);
+      }
```

Comment:
I'm wondering if we shouldn't check the response payload if its not a FHIR Bundle... reason being, as the Sidechain stopped reporting 429 it might also stop reporting 400 and only return the result on the response body. If that happens, we could mark a few keys as rate limited for every malformed XML that the sidechain can't convert.

Reply:
yes good point - need to account for "OperationOutcome" and etc here